src/nnet3/nnet-training.cc

```diff
@@ -235,6 +235,45 @@ void ComputeObjectiveFunction(const GeneralMatrix &supervision,
         computer->AcceptOutputDeriv(output_name, &diff);
       break;
     }
+    case kLognormal: {
```

Comment:
I need to re-align the training data based on tdnn+chain. I am going to use steps/nnet3/align.sh. The problem is that I don't know which lang dir to use. `data/lang_chain_*` or `data/lang` ?
Also what acoustic-scale should I use? The default is `0.1` but the decode dirs in `exp/chain/tdnn_7b_sp` use `1.0`. What about self-loop-scale and transition-scale ?
Thank you